src/domain/index.ts

```diff
@@ -177,3 +177,16 @@ export type ChainTimerWithdraws = Record<HermezEthereumAddress, TimerWithdraw[]>
 
 export type AuthSignatures = Record<ChainId, ChainAuthSignatures>;
 export type ChainAuthSignatures = Record<HermezEthereumAddress, string>;
+
+// User reporting
+export type Message =
```

Comment:
I would call this interface `SnackbarMessage` instead of just `Message`

Reply:
I called this `SnackbarMessage` initially when it was tightened to the snackbar, but now that it's exported from the domain, I don't think it's a good idea to couple it with the snackbar. This models a message to show to the user. It can be shown using a snackbar, a modal window, a dialog, or whatever visual component.

Reply:
Okay, good argument!